pkg/timeutil/timeutil.go

```diff
@@ -20,9 +24,32 @@ func Date(datetime time.Time) time.Time {
 func LastWeekday(weekday time.Weekday) time.Time {
 	date := Date(time.Now())
 
-	for date.Weekday() != weekday {
+	for date.Weekday() !=  weekday{
 		date = date.AddDate(0, 0, -1)
 	}
 
 	return date
 }
+
+// StringToWeekday transforms a string to time.Weekday
+// it returns Monday if wrong day specified in config.toml
+func StringToWeekday(weekday string) (time.Weekday, error){
```

Comment:
something about this method didn't quite sit right with me, it seems like something that golang's `time` should provide, though I have previously commented on how I like `time` for its incredible simplicity. Anyway, I threw together this small test program, not necessarily as a change request, purely as something to think about (mine is probably slower than yours!)

```golang
package main

import (
	"fmt"
	"strings"
	"time"
)

func main() {
	fmt.Println(getWeekday("wednesday"))
	fmt.Println(getWeekday("MONDAY"))
}

func getWeekday(name string) time.Weekday {
	str := strings.Title(strings.ToLower(name))

	for i := 0; i < 7; i++ {
		day := time.Weekday(i)

		if str == day.String() {
			return day
		}
	}

	return time.Sunday
}
```

output:

```
Wednesday
Monday
```